components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -150,42 +134,44 @@
-<config_use_Redi_bottom_layer_tapering>.false.</config_use_Redi_bottom_layer_tapering>
-<config_Redi_bottom_layer_tapering_depth>0.0</config_Redi_bottom_layer_tapering_depth>
-<config_GM_Bolus_kappa_function>'constant'</config_GM_Bolus_kappa_function>
-<config_standardGM_tracer_kappa>1800.0</config_standardGM_tracer_kappa>
-<config_standardGM_tracer_kappa ocn_forcing="datm_forced_restoring" ocn_grid="oEC60to30v3wLI">600.0</config_standardGM_tracer_kappa>
-<config_GM_Bolus_kappa_min>0.0</config_GM_Bolus_kappa_min>
-<config_GM_Bolus_kappa_max>600.0</config_GM_Bolus_kappa_max>
-<config_GM_Bolus_cell_size_min>20e3</config_GM_Bolus_cell_size_min>
-<config_GM_Bolus_cell_size_max>30e3</config_GM_Bolus_cell_size_max>
-<config_Redi_kappa>0.0</config_Redi_kappa>
-<config_gravWaveSpeed_trunc>0.3</config_gravWaveSpeed_trunc>
-<config_max_relative_slope>0.01</config_max_relative_slope>
+<!-- Redi_isopycnal_mixing -->
+<config_use_Redi>.false.</config_use_Redi>
+<config_Redi_kappa>1800.0</config_Redi_kappa>
+<config_Redi_kappa ocn_forcing="datm_forced_restoring">600.0</config_Redi_kappa>
+<config_Redi_maximum_slope>0.3</config_Redi_maximum_slope>
+<config_Redi_closure>'constant'</config_Redi_closure>
+<config_Redi_use_slope_taper>.true.</config_Redi_use_slope_taper>
+<config_Redi_use_surface_taper>.true.</config_Redi_use_surface_taper>
+<config_Redi_N2_limit_term1>.true.</config_Redi_N2_limit_term1>
+
+<!-- GM_eddy_parameterization -->
+<config_use_GM ocn_grid="oQU480">.true.</config_use_GM>
+<config_use_GM ocn_grid="oQU240">.true.</config_use_GM>
+<config_use_GM ocn_grid="oQU240wLI">.true.</config_use_GM>
+<config_use_GM ocn_grid="oQU120">.true.</config_use_GM>
+<config_use_GM ocn_grid="oEC60to30">.true.</config_use_GM>
+<config_use_GM ocn_grid="oEC60to30v3">.true.</config_use_GM>
+<config_use_GM ocn_grid="oEC60to30wLI">.true.</config_use_GM>
+<config_use_GM ocn_grid="oEC60to30v3wLI">.true.</config_use_GM>
+<config_use_GM ocn_grid="oRRS30to10">.false.</config_use_GM>
+<config_use_GM ocn_grid="oRRS30to10v3">.false.</config_use_GM>
+<config_use_GM ocn_grid="oRRS30to10wLI">.false.</config_use_GM>
+<config_use_GM ocn_grid="oRRS30to10v3wLI">.false.</config_use_GM>
+<config_use_GM ocn_grid="oRRS18to6">.false.</config_use_GM>
+<config_use_GM ocn_grid="oRRS18to6v3">.false.</config_use_GM>
+<config_use_GM ocn_grid="oRRS15to5">.false.</config_use_GM>
+<config_use_GM ocn_grid="oARRM60to10">.false.</config_use_GM>
+<config_use_GM ocn_grid="oARRM60to6">.false.</config_use_GM>
+<config_GM_kappa>1800.0</config_GM_kappa>
+<config_GM_kappa ocn_forcing="datm_forced_restoring" ocn_grid="oEC60to30v3wLI">600.0</config_GM_kappa>
+<config_GM_closure>'N2_dependent'</config_GM_closure>
+<config_GM_min_stratification_ratio>0.1</config_GM_min_stratification_ratio>
+<config_GM_constant_gravWaveSpeed>0.3</config_GM_constant_gravWaveSpeed>
+
+<!-- mesoscale_eddy_parameterizations -->
+<config_eddying_resolution_taper>'ramp'</config_eddying_resolution_taper>
```

Comment:
@jonbob this is correct. We can always have `config_eddying_resolution_taper` set to `ramp`. For high resolution, like RRS, GM and Redi are off anyway. For any other mesh, these should taper off when lower than 30km. For EC60to30, it can still be set to ramp with no harm done.